### PR TITLE
Add custom groups scope claim name

### PR DESCRIPTION
**Pull request by Klucherev**
Some OIDC providers don't have "groups" claim in id token, but have "roles". With this setting in traccar.xml there is ability to change it to work properly.

Example is
` <entry key='openid.groupsClaimName'>roles</entry>`

### CODE REVIEW

**Comment by tananaev** [comment]
Which some providers and do you have some documentation references?

**Comment by Klucherev** [comment]
@tananaev i use logto.io as OIDC provider and there is no "groups" claim in token. Only roles with user roles name array

**Comment by tananaev** [comment]
What about documentation?

**Comment by Klucherev** [comment]
Groups claim is not standard claim. Auth0 also have problems to add it. I've read whole documentation of logto.io and there is no such ability. The only thing that i can to do is add custom claim to access token, not id token

File: src/main/java/org/traccar/database/OpenIdProvider.java [hunk 1]
```diff
@@ -111,13 +114,14 @@ public OpenIdProvider(Config config, LoginService loginService, HttpClient httpC
...
         allowGroup = config.getString(Keys.OPENID_ALLOW_GROUP);
+        groupsClaimName = config.getString(Keys.OPENID_GROUPS_CLAIM_NAME);
     }
 
     public URI createAuthUri() {
         Scope scope = new Scope("openid", "profile", "email");
 
         if (adminGroup != null) {
-            scope.add("groups");
+            scope.add(Objects.requireNonNullElse(groupsClaimName, "groups"));
```
Comment (tananaev) [inline src/main/java/org/traccar/database/OpenIdProvider.java:124]: Can we just use `"groups"` as the default value?

Comment (Klucherev) [reply src/main/java/org/traccar/database/OpenIdProvider.java:124]: Yes. Absolutely. requireNonNullElse sets default value if groupsClaimName is not provided.

Comment (tananaev) [reply src/main/java/org/traccar/database/OpenIdProvider.java:124]: Yes, but we have a standard mechanism we use everywhere else.

Comment (Klucherev) [reply src/main/java/org/traccar/database/OpenIdProvider.java:124]: if condition?

Comment (tananaev) [reply src/main/java/org/traccar/database/OpenIdProvider.java:124]: No, configuration keys can have default values.

**Pull request closed without merging**